CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+**0.7.0 - 09/08/23**
+ - Added generate_tax_1040 method
+ - Added copy_from_within_household noise function
+ - Improved performance of typographic noise function
```

Comment:
```suggestion
 - Improved performance of typographic noise type
```
We're not using "noise function" in public facing docs because the functions are not yet public facing.